<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tráº§n Duy Thanh</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/ic_logo.jpg</normaloff>../Images/ic_logo.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Statistics:</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>200</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="title">
             <string>List of Data:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTableWidget" name="tableWidgetStatistic">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>180</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>180</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>180</height>
                </size>
               </property>
               <column>
                <property name="text">
                 <string>Gender</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Number Sales</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Chart Visualization:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayoutPlot"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Functions:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayoutFunctions">
               <item>
                <widget class="QPushButton" name="pushButtonPurchaseRatesByGender">
                 <property name="text">
                  <string>(1) Purchase rates by gender</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_gender.png</normaloff>../Images/ic_statistic_gender.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>From:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditFromAge">
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>30</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>To:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditToAge">
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>50</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonPurchaseRatesByAgeGroup">
                   <property name="text">
                    <string>(2) Purchase rates by age group</string>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../Images/ic_statistic_age.png</normaloff>../Images/ic_statistic_age.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonPurchaseCountingByCategory">
                 <property name="text">
                  <string>(3) Purchase counting by category</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_category.png</normaloff>../Images/ic_statistic_category.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonPurchaseValueByCategory">
                 <property name="text">
                  <string>(4) Purchase value by category</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_cate_value.png</normaloff>../Images/ic_statistic_cate_value.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonPurchaseByCategoryAndGender">
                 <property name="text">
                  <string>(5) Purchases by category and gender</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_cate_gender.png</normaloff>../Images/ic_statistic_cate_gender.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonPaymentMethod">
                 <property name="text">
                  <string>(6) Payment method: Cash, Credit, Debit card</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_payment.png</normaloff>../Images/ic_statistic_payment.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonPurchaseRatesByShoppingMall">
                 <property name="text">
                  <string>(7) Purchase rates by Shopping Mall</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_shopping_mall.png</normaloff>../Images/ic_statistic_shopping_mall.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonProductSpendingByGender">
                 <property name="text">
                  <string>(8) Products spending by gender</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_spending_gender.png</normaloff>../Images/ic_statistic_spending_gender.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonPurchaseFrequenceByAge">
                 <property name="text">
                  <string>(9) Purchase frequency by age</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_frequence_age.png</normaloff>../Images/ic_statistic_frequence_age.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonSalesFluctuationsByMonth">
                 <property name="text">
                  <string>(10) Statistics on sales fluctuations by month</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_sales_monthly.png</normaloff>../Images/ic_statistic_sales_monthly.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonSalesFlucuationsByYearAndMonth">
                 <property name="text">
                  <string>(11) Statistics on sales fluctuations by year and month</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../Images/ic_statistic_sales_yearly_monthly.png</normaloff>../Images/ic_statistic_sales_yearly_monthly.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Machine Learning</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>251</width>
          <height>151</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>INPUT: Independent variables:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QListWidget" name="listWidgetIndependentVariables"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>40</y>
          <width>231</width>
          <height>150</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>TARGET: dependent variables</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QListWidget" name="listWidgetDependentVariables"/>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Test Size:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditTestSize">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>200</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>20</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>200</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Random State:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditRandomState">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>240</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pick Dataset:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxDataset">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonTrainModel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Train Model</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditResult">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>220</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>System</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionConnection"/>
    <addaction name="separator"/>
    <addaction name="actionSave_trained_Machine_Learning_Model"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_trained_Machine_Learning_Model"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInstructions"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSystem"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnection">
   <property name="icon">
    <iconset>
     <normaloff>../Images/ic_connection.png</normaloff>../Images/ic_connection.png</iconset>
   </property>
   <property name="text">
    <string>Connect Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSave_trained_Machine_Learning_Model">
   <property name="icon">
    <iconset>
     <normaloff>../Images/ic_save.png</normaloff>../Images/ic_save.png</iconset>
   </property>
   <property name="text">
    <string>Save trained ML Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_trained_Machine_Learning_Model">
   <property name="icon">
    <iconset>
     <normaloff>../Images/ic_load.png</normaloff>../Images/ic_load.png</iconset>
   </property>
   <property name="text">
    <string>Load trained ML Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../Images/ic_shutdown.png</normaloff>../Images/ic_shutdown.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="icon">
    <iconset>
     <normaloff>../Images/ic_instruction.png</normaloff>../Images/ic_instruction.png</iconset>
   </property>
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../Images/ic_about.png</normaloff>../Images/ic_about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
